pyvista/core/filters/rectilinear_grid.py

```diff
@@ -0,0 +1,80 @@
+"""Filters module with class to manage filters/algorithms for rectilinear grid datasets."""
+
+from pyvista import _vtk, abstract_class
+from pyvista.core.filters import _get_output, _update_alg
+
+
+@abstract_class
+class RectilinearGridFilters:
+    """An internal class to manage filters/algorithms for rectilinear grid datasets."""
+
+    def to_tetrahedra(self, tetra_per_cell=5, mixed=False, pass_cell_ids=False, progress_bar=False):
+        """Create a Tetrahedral mesh from a RectilinearGrid.
+
+        Parameters
+        ----------
+        tetra_per_cell : int, default: 5
+            The number of tetrahedrals to divide each cell in the into. Can be
+            either ``5``, ``6``, or ``12``. If ``mixed=True``, this value is
+            overridden.
+
+        mixed : bool, default: False
```

Comment:
Should we also allow `scalars` to be passed in?  Then we would activate them here.  What if there are no active scalars?

Reply:
Added in 8bcb10af7417fc5ab491b6922f193f7d3fd64ae5.